AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -82,12 +75,12 @@ conda activate user_tensorflow
 
 Navigate to the Model Zoo for Intel® Architecture source directory. By default, it is in your installation path, like `/opt/intel/oneapi/modelzoo`. 
 
-1. View the available Model Zoo release versions for the AI Kit:
+1. View the available Intel® Reference models release versions for the AI Tools:
    ```
    ls /opt/intel/oneapi/modelzoo
-   2.11.0  latest
+   2.13.0  latest
    ```
-2. Navigate to the [Model Zoo Scripts](https://github.com/IntelAI/models/tree/v2.11.0/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
+2. Navigate to the [Intel® Reference models Scripts](https://github.com/IntelAI/models/tree/v2.11.0/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
    ```
    cd /opt/intel/oneapi/modelzoo/latest
```

Comment:
`/opt/intel/oneapi/reference_models`

Reply:
addressed it accordingly. /opt/intel/oneapi/reference_models/latest